rpmlint/pkg.py

```diff
@@ -19,8 +19,9 @@
 except ImportError:
     _magic = None
 import rpm
-from rpmlint.helpers import byte_to_string, print_warning
 
+from .helpers import byte_to_string, print_warning
+from .pkgfile import PkgFile
```

Comment:
please either do not use relative imports or switch even other files to use them.